### PR TITLE
Adds kale smoothies to foods list

### PR DESCRIPTION
**Pull request by KipHamiltons**
Examples of things worth having in PR descriptions:
This PR solves this issue (link), and passes these tests, but I'm not sure if it fits in with what we're doing here with this module.

### CODE REVIEW

File: Foods.yaml [hunk 1]
```diff
@@ -6,5 +6,4 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["Pizza"]
-  
\ No newline at end of file
+  - ["Pizza", "kale_smoothies"]
```
Comment (ysims) [inline Foods.yaml]: Why are these different styles? We doing camel case or snake case?

Comment (TrentHouliston) [reply Foods.yaml]: We should try to stick to `snake_case` for yaml files in the NUbots codebase(s)!

Comment (KipHamiltons) [reply Foods.yaml]: good point!

**Review by ysims — APPROVED** [review]
LGTM

**Pull request merged**